synphot/filter_parameterization/tests/data/fft_test_data.fits

```diff
@@ -0,0 +1,7 @@
                                                  TTYPE14 = 'fft_8   '                                                            TFORM14 = 'M       '                                                            TTYPE15 = 'fft_9   '                                                            TFORM15 = 'M       '                                                            END                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             SLOAN/SDSS.u        /E:@ A�  =�5?@cS�@         ��r��P����"� ��?��>
```

Comment:
I haven't fully thought through the implications, but would it make sense to store these data as something other than a FITS file? Perhaps a CSV file or JSON would be more diff-able?

Reply:
FITS file size is smaller though. 🤔